src/get-started/data-proofs.md

```diff
@@ -41,32 +41,33 @@ Add necessary dependencies to `Cargo.toml` in the project directory:
 
 ```toml
 [dependencies]
-exonum = "0.9.0"
-exonum-configuration = "0.9.0"
-serde = "1.0"
-serde_derive = "1.0"
-failure = "=0.1.1"
+exonum = "0.10.2"
+exonum-derive = "0.10.0"
+exonum-configuration = "0.10.1"
+serde = "1.0.0"
+serde_derive = "1.0.0"
+failure = "0.1.5"
+protobuf = "2.2.0"
 ```
 
-Then import said dependencies into `src/lib.rs` file. We will use them to fetch
-the necessary structures, functions, types, traits etc.:
+Then we need to import macro from dependencies into `src/lib.rs` file.
 
 ```rust
 #[macro_use]
-extern crate exonum;
+extern crate exonum_derive;
 #[macro_use]
 extern crate failure;
```

Comment:
What do you think about to remove redundant `extern crate...`s ? Just because `Rust 2018` ?

Reply:
It is impossible to do this for serde since it is used in protobuf generated files, otherwise ok.